### ISSUE TITLE
improve accessible navigation: use headings

### ISSUE THREAD

**Issue by sharafeddeen** [StatisticsHelper/statistics-helper#9]
Headings is another way that users navigate the webpage. Just like landmarks, headings are very important in the user experience. 
There should be only one `h1` heading per page (because it’s not a blog page), so everything else should be `h2`, `h3`, etc. 
Use the Chrome extension `Heading Maps` to understand the layout of the headings on the webpage. This helps users that use this layout to navigate the page. https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en
